cli/cmd/project/hibernate.go

```diff
@@ -21,7 +21,8 @@ func HibernateCmd(ch *cmdutil.Helper) *cobra.Command {
 			if len(args) > 0 {
 				project = args[0]
 			}
-			if !cmd.Flags().Changed("project") && len(args) == 0 && ch.Interactive {
+			// Only prompt interactively if no flags or args are provided
+			if !cmd.Flags().Changed("project") && len(args) == 0 && ch.Interactive && !cmd.Flags().Changed("path") {
```

Comment:
Should be reverted – it should still proceed to inference even if `path` is changed (in fact, the whole point of `path` is to support inference)